src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -48,8 +48,6 @@ const UsersFragment = ({
 
   const [myId, setMyId] = React.useState('');
 
-  const isReady = useIsReady();
-
   const users = React.useMemo(() => {
     return discovery.initialUsers.map((item) => ({
       ...item,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isReady` variable has been removed. If this variable was used to check if certain conditions were met before rendering the component, its removal could lead to unexpected behavior or errors. Please ensure that the removal of `isReady` does not affect the functionality of the component.
```diff
- const isReady = useIsReady();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->